client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/utils.ts

```diff
@@ -33,7 +33,7 @@ export const issueStock = (
     ({ id }) => id === idToIssue
   );
   const foundRow = draftOutboundLines[foundRowIdx];
-  if (!foundRow) return [];
+  if (!foundRow) return draftOutboundLines;
```

Comment:
bonus bug - sometimes when updating individual row quantities, the table would disappear and I'd get the 'no rows' message. This might be the problem - I couldn't replicate after isolating it to this general area.